Crashlytics/CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 
+# v4.0.0-beta.5
+
+- [changed] Updates two endpoints in the SDK with no expected end-user impact (#4953, #4988)
```

Comment:
"Changed"

Reply:
These tags are consumed by a script and are supposed to be lowercase. They'll be imported to devsite as `* {{ changed }} Updates two endpoints ...`

Reply:
Oops, looks like I misread your comment. 👍

Reply:
I think it was in reference to "Updates two..." => "Changed two..."